.CMake/alg_support.cmake

```diff
@@ -433,7 +433,7 @@ if(NOT ((OQS_MINIMAL_BUILD STREQUAL "") OR (OQS_MINIMAL_BUILD STREQUAL "OFF")))
IG_slh_dsa_sha2_512_256_prehash_shake_192f;SIG_slh_dsa_sha2_512_256_prehash_shake_256s;SIG_slh_dsa_sha2_512_256_prehash_shake_256f;SIG_slh_dsa_sha3_224_prehash_sha2_128s;SIG_slh_dsa_sha3_224_prehash_sha2_128f;SIG_slh_dsa_sha3_224_prehash_sha2_192s;SIG_slh_dsa_sha3_224_prehash_sha2_192f;SIG_slh_dsa_sha3_224_prehash_sha2_256s;SIG_slh_dsa_sha3_224_prehash_sha2_256f;SIG_slh_dsa_sha3_224_prehash_shake_128s;SIG_slh_dsa_sha3_224_prehash_shake_128f;SIG_slh_dsa_sha3_224_prehash_shake_192s;SIG_slh_dsa_sha3_224_prehash_shake_192f;SIG_slh_dsa_sha3_224_prehash_shake_256s;SIG_slh_dsa_sha3_224_prehash_shake_256f;SIG_slh_dsa_sha3_256_prehash_sha2_128s;SIG_slh_dsa_sha3_256_prehash_sha2_128f;SIG_slh_dsa_sha3_256_prehash_sha2_192s;SIG_slh_dsa_sha3_256_prehash_sha2_192f;SIG_slh_dsa_sha3_256_prehash_sha2_256s;SIG_slh_dsa_sha3_256_prehash_sha2_256f;SIG_slh_dsa_sha3_256_prehash_shake_128s;SIG_slh_dsa_sha3_256_prehash_shake_128f;SIG_slh_dsa_sha3_256_prehash_shake_192s;SIG_slh_dsa_sha3_256_prehash_shake_192f;SIG_slh_dsa_sha3_256_prehash_shake_256s;SIG_slh_dsa_sha3_256_prehash_shake_256f;SIG_slh_dsa_sha3_384_prehash_sha2_128s;SIG_slh_dsa_sha3_384_prehash_sha2_128f;SIG_slh_dsa_sha3_384_prehash_sha2_192s;SIG_slh_dsa_sha3_384_prehash_sha2_192f;SIG_slh_dsa_sha3_384_prehash_sha2_256s;SIG_slh_dsa_sha3_384_prehash_sha2_256f;SIG_slh_dsa_sha3_384_prehash_shake_128s;SIG_slh_dsa_sha3_384_prehash_shake_128f;SIG_slh_dsa_sha3_384_prehash_shake_192s;SIG_slh_dsa_sha3_384_prehash_shake_192f;SIG_slh_dsa_sha3_384_prehash_shake_256s;SIG_slh_dsa_sha3_384_prehash_shake_256f;SIG_slh_dsa_sha3_512_prehash_sha2_128s;SIG_slh_dsa_sha3_512_prehash_sha2_128f;SIG_slh_dsa_sha3_512_prehash_sha2_192s;SIG_slh_dsa_sha3_512_prehash_sha2_192f;SIG_slh_dsa_sha3_512_prehash_sha2_256s;SIG_slh_dsa_sha3_512_prehash_sha2_256f;SIG_slh_dsa_sha3_512_prehash_shake_128s;SIG_slh_dsa_sha3_512_prehash_shake_128f;SIG_slh_dsa_sha3_512_prehash_shake_192s;SIG_slh_dsa_sha3_512_prehash_shake_192f;SIG_slh_dsa_sha3_512_prehash_shake_256s;SIG_slh_dsa_sha3_512_prehash_shake_256f;SIG_slh_dsa_shake_128_prehash_sha2_128s;SIG_slh_dsa_shake_128_prehash_sha2_128f;SIG_slh_dsa_shake_128_prehash_sha2_192s;SIG_slh_dsa_shake_128_prehash_sha2_192f;SIG_slh_dsa_shake_128_prehash_sha2_256s;SIG_slh_dsa_shake_128_prehash_sha2_256f;SIG_slh_dsa_shake_128_prehash_shake_128s;SIG_slh_dsa_shake_128_prehash_shake_128f;SIG_slh_dsa_shake_128_prehash_shake_192s;SIG_slh_dsa_shake_128_prehash_shake_192f;SIG_slh_dsa_shake_128_prehash_shake_256s;SIG_slh_dsa_shake_128_prehash_shake_256f;SIG_slh_dsa_shake_256_prehash_sha2_128s;SIG_slh_dsa_shake_256_prehash_sha2_128f;SIG_slh_dsa_shake_256_prehash_sha2_192s;SIG_slh_dsa_shake_256_prehash_sha2_192f;SIG_slh_dsa_shake_256_prehash_sha2_256s;SIG_slh_dsa_shake_256_prehash_sha2_256f;SIG_slh_dsa_shake_256_prehash_shake_128s;SIG_slh_dsa_shake_256_prehash_shake_128f;SIG_slh_dsa_shake_256_prehash_shake_192s;SIG_slh_dsa_shake_256_prehash_shake_192f;SIG_slh_dsa_shake_256_prehash_shake_256s;SIG_slh_dsa_shake_256_prehash_shake_256f")
```

Comment:
@dstebila Do we really want to include all 156 variants? I am thinking of including only the 12 "pure" variants.